cx/opcodes_core.go

```diff
@@ -264,6 +264,10 @@ const (
 	OP_F32_NEG
 	OP_F64_NEG
 	END_OF_CORE_OPS
+
+	// http
+	OP_HTTP_SERVE
+	OP_HTTP_NEW_REQUEST
```

Comment:
This is not working correctly. Error for making the call for func behind OP_HTTP_NEW_REQUEST is as follows:
`CX_RUNTIME_ERROR, multiple opcode handler finders returned an opcode handler`

But when position with OP_HTTP_SERVE is swapped to:
`OP_HTTP_NEW_REQUEST 
OP_HTTP_SERVE `
Then the same error is returned for OP_HTTP_SERVE.

Also, when I introduce something like following code neither of commands is working. Meaning that for some reason commands are not handled correctly after certain amount:
```
OP_NOP // dummy command just to increase the number of commands before http ones

// http
OP_HTTP_SERVE
OP_HTTP_NEW_REQUEST
```
>